programs/Makefile

```diff
@@ -88,6 +88,12 @@ ifneq (,$(filter x86_64%,$(shell $(CC) -dumpmachine)))
 else
     RES_FILE = $(RES32_FILE)
 endif
+RC ?= windres
+# http://stackoverflow.com/questions/708238/how-do-i-add-an-icon-to-a-mingw-gcc-compiled-executable
+$(RES64_FILE): windres/zstd.rc
```

Comment:
A rule in first position becomes the default rule (triggered when invoking `make` with no argument).

Currently, it's `zstd`, but with this patch, it becomes `$(RES64_FILE)`.

Possible work-around : add a `default` rule which delegates to `zstd`
early enough in the `Makefile` to ensure it is the first rule.

Reply:
I'll move it down